src/main/java/org/spongepowered/api/item/inventory/Carrier.java

```diff
@@ -36,6 +34,6 @@
      *
      * @return The inventory this Carrier is holding
      */
-    CarriedInventory<? extends Carrier> getInventory();
+    Inventory getInventory();
```

Comment:
You shouldn't really change this because you may return an inventory from a random block entity